addon/list-utils.js

```diff
@@ -0,0 +1,69 @@
+import * as utils from './utils'
+import Ember from 'ember'
+
+function promiseValue (value) {
+  return new Ember.RSVP.Promise(function (resolve) {
+    resolve(value)
+  })
+}
+
+/**
+ * setOptions - set a list's available options
+ * @param  {Object} modelDef the bunsen model definition
+ * @param  {Object} dbStore the ember-data store
+ * @returns {Promise} a promise that resolves to a list of items
+ */
+export function getOptions (modelDef, dbStore) {
+  const enumDef = modelDef.enum
+  const queryDef = modelDef.query
+  let result
+  if (enumDef) {
+    result = promiseValue(getEnumValues(enumDef))
+  } else if (queryDef) {
+    result = getAsyncDataValues(modelDef, queryDef, dbStore)
+  } else {
+    result = promiseValue([])
+  }
+  return result
+}
+
+/**
+ * getEnumValues - take enum values and make into a collection
+ * @param  {String[]} values the values to make into a collection
+ * @returns {Object[]} a list of item objects
+ */
+export function getEnumValues (values = []) {
+  return values.map((value) => {
+    const label = Ember.String.capitalize(
+      value.split('-').join(' ').toLowerCase()
+    )
+    return {label, value}
+  })
+}
+
+/**
+ * setAsyncDataValues - Fetch the list of network functions from the backend and set them
```

Comment:
Here's an example of why you don't want to put the name of the function in the description. Looks like you've got the wrong name, and apparently that's not checked with our `eslint` rule for `valid-jsdoc`